manim/scene/scene.py

```diff
@@ -65,7 +65,10 @@ def construct(self):
     def __init__(self, **kwargs):
         Container.__init__(self, **kwargs)
         self.camera = self.camera_class(**camera_config)
-        self.file_writer = SceneFileWriter(self, **file_writer_config,)
+        self.file_writer = SceneFileWriter(
+            self,
+            **file_writer_config,
+        )
```

Comment:
Comment: This is only happening because of the trailing comma. I believe if you delete the trailing comma, black will just shut up about this line.